optima/calibration.py

```diff
@@ -181,7 +186,7 @@ def convert(pars, parlist, parvec=None):
     
 
 
-    def errorcalc(parvec=None, pars=None, parlist=None, project=None, bestindex=0):
+    def errorcalc(parvec=None, pars=None, parlist=None, project=None, fitto='prev', bestindex=0, doplot=False, verbose=2):
```

Comment:
Eventually we could allow users could choose the GOF criteria, that would be amazing.